### PR TITLE
BaSP-MR-09-16: Post Trainers

### PR DESCRIPTION
**Pull request by Flor-Farruggia**
Request for create a new trainer user.

### CODE REVIEW

File: src/controllers/trainer.js [hunk 1]
```diff
@@ -73,4 +73,31 @@ const updateTrainer = (req, res) => {
...
+    .catch((error) => {
+      res.status(500).json({
+        message: 'Trainer cannot be created',
+        error,
+      });
+    });
+};
+
+module.exports = {
+  getAllTrainers, getTrainerById, updateTrainer, postTrainer,
```
Comment (danielninfi) [inline src/controllers/trainer.js:102]: this generated a problem for me (it does not mean that it generated it for you).
to solve it the recommended identation is 
module.exports = {
 getAllTrainers, 
getTrainerById, 
updateTrainer, 
deleteTrainer,
};

**Review by danielninfi — CHANGES_REQUESTED** [review]
in postman your trainer files are loose and not in a container folder

**Review by OscarLucero — CHANGES_REQUESTED** [review]
In postman you didnt create the folder for trainer to put the get files to export in the correct way. I did it in the pr of delete and added yours to the folder, keep an eye on that.

File: src/controllers/trainer.js (hunk 1 above)
Comment (Flor-Farruggia) [reply src/controllers/trainer.js:102]: genial! ahora veo si lo puedo arreglar, porque eso me lo acomodó así el visual, pero bueno saber!

**Review by NicoCagnina — APPROVED** [review]

File: src/routes/trainer.js [hunk 2]
```diff
@@ -1,12 +1,14 @@
 const express = require('express');
 const { validateUpdate } = require('../validations/trainer');
 const trainerController = require('../controllers/trainer');
+const validation = require('../validations/trainer');
 
 const router = express.Router();
 
 router
   .put('/:id', validateUpdate, trainerController.updateTrainer)
```
Comment (JuanMornik) [inline src/routes/trainer.js:9]: replace validateUpdate with valdiation.validateUpdate since it's now inside an object

**Review by JuanMornik — CHANGES_REQUESTED** [review]

**Review by OscarLucero — APPROVED** [review]

**Review by danielninfi — APPROVED** [review]

**Pull request merged**